src/Tribe/Aggregator/Admin_Bar.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Class to initialize and add the required items in
+ *
+ * @since  TBD
+ */
+class Tribe__Events__Aggregator__Admin_Bar {
+
+	/**
+	 * @since  TBD
+	 *
+	 * @var Tribe__Events__Admin__Bar__Admin_Bar
+	 */
+	protected static $instance;
+	/**
+	 * @since  TBD
+	 *
+	 * @var Tribe__Events__Constants
+	 */
+	protected $constants;
+	/**
+	 * @since  TBD
+	 *
+	 * @var Tribe__Events__Admin__Bar__Configurator_Interface
+	 */
+	protected $config;
+
+	/**
+	 * Singleton constructor for the class.
+	 *
+	 * @since  TBD
+	 *
+	 * @return Tribe__Events__Aggregator__Admin_Bar
+	 */
+	public static function instance() {
```

Comment:
Use di52 rather than a static singleton?

Reply:
For the most part we don't have di52 on Aggregator, so I would rather not do it on this MR.